src/lib/maplibre/controls/CustomControl/Icon.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+	import type { Snippet } from 'svelte';
+	let {
+		children,
+		onclick,
+		oncontextmenu,
+		className
```

Comment:
This is an idiom I frequently come across!

```suggestion
		class: className
```